flask_project/campaign_manager/models/models.py

```diff
@@ -307,6 +324,34 @@ def get_participants(self):
         campaign."""
         return len(self.users)
 
+    def get_task_boundary_coordinates(self):
+        """ Returns the coordinates of the TaskBoundary. """
+        coordinates = self.get_task_boundary_as_geoJSON()
+        coordinates = ast.literal_eval(coordinates[0])
+        coordinates = coordinates['coordinates'][0]
+        return coordinates
+
+    def get_participant_count(uuid):
+        """ Returns the participant count for the campaign. """
+        participant_count = 0
+        campaign = session.query(Campaign).filter(
+            Campaign.uuid == uuid).fisrt()
+        # add all managers to the participant count
+        participant_count += len(campaign.users)
+        # add all team members to the participant count
+        for task_boundary in campaign.task_boundaries:
+            participant_count += len(task_boundary.teams.users)
+        return participant_count
+
+    def update_participants_count(self, participants_count, campaign_type):
+        """ Updates the participant count of individual campaign feature. """
+        feature_type = session.query(FeatureType).filter(
+            and_(
+                FeatureType.name == campaign_type,
+                FeatureType.campaign_id == self.id))
+        feature_type.member_count += participant_count
```

Comment:
`member_count` or `participant_count`?
We should keep the same name.
=> participant_count

Reply:
As of now the member_count, counts all the members that are registered for the campaign and with version-2(database app) but the osm data is might be collected by the members who are not the part of campaign. The participant_count counts that , all the users that have collected the data for the geolocation irrespective of them being part of the campaign.
Maybe @timlinux @smit1678 can provide us some more information. In the current deployed application there is no method to create a team, what I think is that member count refers to registered members and participant count to all members(registered and non registered).

Or another method could be to get the list of all the members for a geolocation from overpass based on the feature type and add them to the application, then maybe we can have a single variable for count. But this is a task big enough in itself.
what do you say.